yt/loaders.py

```diff
@@ -1344,7 +1344,7 @@ def load_sample(fn=None, progressbar: bool = True, timeout=None, **kwargs):
     if not isinstance(specs["load_kwargs"], dict):
         raise ValueError(
             "The requested dataset seems to be improperly registered.\n"
-            "Tip: the entry in yt/sample_data_registry.json may inconstistent with "
+            "Tip: the entry in yt/sample_data_registry.json may be inconststent with "
```

Comment:
```suggestion
            "Tip: the entry in yt/sample_data_registry.json may be inconsistent with "
```

Reply:
Damn how can I get this wrong _twice_ ? Thanks for picking it